WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignListViewModel.swift

```diff
@@ -11,6 +12,16 @@ extension BlazeCampaign: Identifiable {
 
 /// View model for `BlazeCampaignListView`
 final class BlazeCampaignListViewModel: ObservableObject {
+    enum CreateCampaignDestination: Equatable {
+        case productSelector
+        case campaignForm(productID: Int64)
+        case webviewForm(productID: Int64?)
```

Comment:
Super nit: 
```suggestion
        case webViewForm(productID: Int64?)
```

Reply:
This is handled in the new Coordinator class https://github.com/woocommerce/woocommerce-ios/pull/11671/files#diff-97e95c754013a21f67871677e78df49649e1cd07195b4f1a5f3128a19ac2516aR11